### PR TITLE
Initial contributing info.

### LINKED ISSUE #37: Need information describing the contributing guidelines

### PR DESCRIPTION
**Pull request by markemeis**
Fixes #37 

Signed-off-by: Mark Emeis <mark.emeis@broadcom.com>

### CODE REVIEW

**Review by D3nn — APPROVED** [review]
LGTM

File: reporting_bugs.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# Reporting bugs
+
+If any part of the yipee project has bugs or documentation mistakes, please let us know by [opening an issue](https://github.com/yipeeio/yipee/issues). We treat bugs and mistakes very seriously and believe no issue is too small. Before creating a bug report, please check that an issue reporting the same problem does not already exist.
```
Comment (jrryjcksn) [inline reporting_bugs.md:3]: URL is now incorrect

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
+# Contributing to yipee
+
+Welcome! We're excited that you're interested in contributing. 
+
+Below you will find some basic steps required to be able to contribute to the project. If you have any questions about this process or any other aspect of contributing to a Yipee open source project, please open an [issue](https://github.com/yipeeio/yipee/issues) and we'll get your questions answered as quickly as we can.
+
+Pull Requests are always welcome, however they will only be accepted if they provide a reasonable test coverage.
```
Comment (jd3quist) [inline CONTRIBUTING.md]: Either "provide reasonable test coverage" or "provide a reasonable amount of test coverage"

**Review by jrryjcksn — APPROVED** [review]
Good stuff.

File: reporting_bugs.md (hunk 1 above)
Comment (jd3quist) [inline reporting_bugs.md:3]: Seems like https://github.com/yipeeio/yipee/issues isn't gonna be the right URL

**Review by jd3quist — APPROVED** [review]
Just a couple nits

File: reporting_bugs.md (hunk 1 above)
Comment (markemeis) [reply reporting_bugs.md:3]: Changed URL to be relative.

File: CONTRIBUTING.md (hunk 2 above)
Comment (markemeis) [reply CONTRIBUTING.md]: Fixed the typo.

**Pull request merged**